server/usecases/contact/inquiry.go

```diff
@@ -0,0 +1,26 @@
+package contact
+
+import (
+	"database/sql"
+
+	"github.com/h3poteto/fascia/lib/modules/database"
+	entity "github.com/h3poteto/fascia/server/domains/entities/inquiry"
+	repo "github.com/h3poteto/fascia/server/infrastructures/inquiry"
+)
+
+func InjectDB() *sql.DB {
+	return database.SharedInstance().Connection
+}
+
+func InjectInquiryRepository() entity.InquiryRepository {
+	return repo.New(InjectDB())
+}
+
+func CreateInquiry(email, name, message string) (*entity.Inquiry, error) {
```

Comment:
__[Golint]__ exported function CreateInquiry should have comment or be unexported

<sub>posted by Sider</sub>